documentation/tutorials/getting-started-with-ash-and-phoenix.md

```diff
@@ -0,0 +1,520 @@
+# Get Started with Ash and Phoenix
+
+<!--- ash-hq-hide-start --> <!--- -->
+
+This documentation is best viewed at [ash-hq.org](https://ash-hq.org)
+
+<!--- ash-hq-hide-stop --> <!--- -->
+
+## Who is This For?
+
+This is designed to be a quick start guide for Ash with Phoenix. Familiarity with Phoenix and LiveView are not necessary, but would certainly be helpful.
+
+## Goals
+
+In this guide we will:
+
+1. Create a new Phoenix project
+2. Setup Ash, AshPhoenix and AshPostgres as dependencies
+3. Create a basic `Blog.Post` resource
+4. Create and migrate the database
+5. Learn how to interact with your resource
+6. Integrate a minimal Phoenix LiveView with Ash
+
+## Things You May Want to Read First
+
+- [Install Elixir](https://elixir-lang.org/install.html)
+- [Phoenix - Up and Running Guide](https://hexdocs.pm/phoenix/up_and_running.html)
+- [Philosophy Guide](/documentation/tutorials/philosophy.md)
+
+## Requirements
+
+If you want to follow along yourself, you will need the following things:
+
+1. Elixir (1.12 or later) and Erlang (22 or later) installed
+2. PostgreSQL installed
+3. A text editor
+4. A terminal to run the examples
+
+## Setup
+
+### Create a New Phoenix Project
+
+_This section is based on the [Phoenix installation docs](https://hexdocs.pm/phoenix/installation.html). For more details go there._
+
+First we need to install the Phoenix project generator, then we'll run the generator to create our new project.
+
+**NOTE: DO NOT run `mix ecto.create`, (as it asks you to) we will do this the Ash way later.**
+
+```bash
+# install Phoenix project generator
+$ mix archive.install hex phx_new
+
+# generate Phoenix project
+$ mix phx.new my_ash_phoenix_app
+
+# cd into project
+$ cd my_ash_phoenix_app
+```
+
+### Add Dependencies
+
+We now need to add Ash, AshPhoenix and AshPostgres to our Phoenix project. We need to add the dependencies to the `deps` function in our `mix.exs`. We'll also need to add dependencies to our `.formatter.exs` to ensure consistent formatting when using `mix format`.
+
+```elixir
+# mix.exs
+
+  def deps do
+    [
+      {:phoenix, "~> 1.6.15"},
```

Comment:
Can you make this `"~> x.x"` and then put a note that they can use `mix hex.info library_name` to get the latest version? We just had a user get confused recently by a guide with hard coded versions and not being on the latest verisons.

Reply:
For all of the dependencies, or just this one?

Reply:
addressed: [1cd3c7c](https://github.com/ash-project/ash_phoenix/pull/70/commits/1cd3c7cf20190949968d571307171b670053bf3b)